staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/fifo_list.go

```diff
@@ -48,6 +48,10 @@ type fifo interface {
 	// Length returns the number of requests in the list.
 	Length() int
 
+	// Width returns the total width (number of seats) of requests
+	// in this list.
+	Width() int
```

Comment:
"Width" is a little awkward here, since the requests are generally not all run at the same time, so the sum of their widths is not a number of seats that gets occupied at once.  How about `SumOfSeats` or `SeatsSum`?

Reply:
SGTM - I opened https://github.com/kubernetes/kubernetes/pull/103534